.pre-commit-config.yaml

```diff
@@ -37,3 +37,9 @@ repos:
     hooks:
       - id: yamllint
         files: '^.github/'
+  - repo: https://github.com/psf/black
+    rev: '20.8b1'
```

Comment:
Is this suppose to be different than the one specified in requirements.txt (```20.10.0```)?

Reply:
It'd be nice if there was a way to pull the revision from the requirements file instead of duplicating it, but that's just me being a big fan of data deduplication.

Reply:
it'd be nice but playing devil's advocate this is a git revision whereas the other one is pypi package version maybe they wouldn't always map 1to1